### PR TITLE
Making some updates to the github readme

### PR DESCRIPTION
**Pull request by bedroesb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
 
-If you already have Ruby installed on your machine, then the following files will allow you to get started by installing the relevant dependencies:
+### Previewing your changes on your own fork
+
+Since the website uses [GitHub pages](https://pages.github.com/) for its deployment, it is possible to create live previews to inspect your changes. Go to your fork -> settings tab -> pages and select the branch you are working on to activate GitHub pages. GitHub will do the rest and will tell you at which url the website will be served. 
+
+### Deploying the website locally
+
+It is possible to preview the website locally to preview changes. We recommand installing Ruby version 2.7.3 or earlier since recent version are not compatible with Jekyll. If you already have the correct Ruby installed on your machine, then the following files will allow you to get started by installing the relevant dependencies:
```
Comment (AlasdairGray) [inline README.md]: Is this specific to the GitHub use of Jekyll, since the Jekyll site itself talks about using Ruby version 3.0?

**Review by AlasdairGray — APPROVED** [review]
Thanks for tidying this up

File: README.md (hunk 1 above)
Comment (bedroesb) [reply README.md]: you have a good point! I never made it to work a year ago. I looked into it and it seems that you need webrick as a gem as part of your project. https://github.com/jekyll/jekyll/issues/8523

Comment (bedroesb) [reply README.md]: https://stackoverflow.com/a/67994527

**Pull request merged**